packages/es-components/src/components/containers/notification/useNotification.js

```diff
@@ -50,7 +60,9 @@ const NotificationContent = React.forwardRef(
 
     return (
       <>
-        {includeIcon ? <NotificationIcon name={iconName} size={28} /> : null}
+        {includeIcon ? (
+          <NotificationIcon name={iconName} iconColor={iconColor} size={28} />
+        ) : null}
```

Comment:
We could use a `&&` operator here instead of the ternary.

Reply:
Good catch, I much prefer this over a ternary. Changes made.